apps/builder/app/builder/features/style-panel/property-label.tsx

```diff
@@ -326,7 +326,7 @@ export const PropertyInlineLabel = ({
 }: {
   label: string;
   description: string;
-  properties: [StyleProperty, ...StyleProperty[]];
+  properties: [StyleProperty, ...StyleProperty[]] | string;
```

Comment:
its confusing, properties is plural, that should be an array ...  even if inside its only one element

Reply:
If i tried to keep `[StyleProeprty | string, ...StyleProperty[] | string]` it's breaking auto-complete. Found this might be a better way to differentiate those both.